packages/pluggableWidgets/datagrid-web/tests/e2e/specs/DataGrid.spec.ts

```diff
@@ -0,0 +1,93 @@
+import page from "../../../../../../configs/e2e/src/pages/page";
+
+describe("datagrid-web", () => {
+    beforeAll(() => {
+        page.open(); // resets page
+    });
+
+    describe("capabilities: sorting", () => {
+        it("changes order of data to ASC when clicking sort option", () => {
+            const datagrid = page.getWidget("datagrid1");
+            datagrid.waitForExist();
+            const column = page.getElement(".column-header*=First Name", datagrid);
+            const icon = page.getElement("svg", column);
+            column.click();
+            expect(icon.getAttribute("data-icon")).toContain("long-arrow-alt-up");
+        });
+        it("changes order of data to DESC when clicking sort option", () => {
+            const datagrid = page.getWidget("datagrid1");
+            const column = page.getElement(".column-header*=First Name", datagrid);
+            const icon = page.getElement("svg", column);
+            datagrid.waitForExist();
+            column.click();
+            expect(icon.getAttribute("data-icon")).toContain("long-arrow-alt-down");
+        });
+    });
+    describe("capabilities: filtering", () => {
+        it("changes the result showed in the data grid following the filter inserted", () => {});
```

Comment:
What's going on with this empty spec, is this functionality still in development?

Reply:
My bad, I forgot to add my changes here... 😔

Reply:
Removed these filtering describe. It will be added a new spec for each widget filter.